### PR TITLE
Constrain input stripe height of lrf to fit in frame

### PR DESCRIPTION
**Pull request by KyleSiefring**
Might fix an rdo problem with odd frame sizes. No tests were run on
this.

### CODE REVIEW

File: src/lrf.rs [hunk 1]
```diff
@@ -964,11 +966,12 @@ impl RestorationState {
       for si in 0..stripe_n {
         let (stripe_start_y, stripe_size) = if si == 0 {
           (0, (64 - 8) >> ydec)
-        }  else {
+        } else {
+          let start = (si * 64 - 8) >> ydec;
           (
-            (si as isize * 64 - 8) >> ydec,
+            start as isize,
             // one past, unlike spec
```
Comment (KyleSiefring) [inline src/lrf.rs]: I'm unsure what this comment means.

**Comment by KyleSiefring** [comment]
https://beta.arewecompressedyet.com/?job=master-193cf46fa97f3c6cd872861add23409f57221236&job=test_height_bounds_lrf1%402019-07-19T18%3A58%3A27.911Z

File: src/lrf.rs (hunk 1 above)
Comment (xiphmont) [reply src/lrf.rs]: I'm using a C-style fencepost so you can check it against foo < end unlike the spec where they use foo <= end

Comment (xiphmont) [reply src/lrf.rs]: The spec uses end fenceposts set to the last valid entry and checks that accesses are <= that value.
I used a C-style fencepost value that's one past the end and check that accesses are < that value.

**Pull request merged**